tests/ui/invalid-compile-flags/emit-output-types-without-args.rs

```diff
@@ -0,0 +1 @@
+//@ compile-flags: --emit
```

Comment:
Suggestion: an add `//@ error-pattern` to ensure that the stderr can't be easily accidentally blessed away, i.e.

```
//@ compile-flags: --emit
//@ error-pattern: Argument to option 'emit' missing
```

Normally you'd use `//~?` but `--emit` doesn't respect `--error-format=json`, cf. #140101.